api/envoy/config/overload/v3/overload.proto

```diff
@@ -175,6 +199,9 @@ message OverloadManager {
   // The set of overload actions.
   repeated OverloadAction actions = 3;
 
+  // The set of load shed points.
```

Comment:
If this is currently not really usable (no actual Load-shed points), consider tagging this as not-implemented-hide, to reduce confusion until some Load-Shed points are available.

Reply:
done